nipype/interfaces/base.py

```diff
@@ -123,10 +123,10 @@ class Bunch(object):
     --------
     >>> from nipype.interfaces.base import Bunch
     >>> inputs = Bunch(infile='subj.nii', fwhm=6.0, register_to_mean=True)
-    >>> inputs # doctest: +ALLOW_UNICODE
+    >>> inputs
```

Comment:
why remove unicode?

```
Expected:
    Bunch(fwhm=6.0, infile='subj.nii', register_to_mean=True)
Got:
    Bunch(fwhm=6.0, infile=u'subj.nii', register_to_mean=True)
```

Reply:
I set this in `pytest.ini`: https://github.com/nipy/nipype/blob/master/pytest.ini#L4

is it failing?

Reply:
You'd have to show me where it's failing. It works on my py2 and all tests for PR were also passing